pyccel/ast/headers.py

```diff
@@ -89,11 +89,33 @@ def __getnewargs__(self):
         args = (self._args[0],)
         return args
 
+#==============================================================================
+class Template(Header):
+    """Represents a Template."""
+
+    def __new__(cls, *args, **kwargs):
+        return Basic.__new__(cls)
+
+    def __init__(self, name, args):
+        Header.__init__(self)
+        self._name = name
+        self._args = args
+
+    @property
+    def name(self):
+        "The name of the template."
+        return self._name
+
+    @property
+    def args(self):
+        "Types the template represents."
+        return self._args
+
 #==============================================================================
 class FunctionHeader(Header):
     """Represents function/subroutine header in the code.
 
-    func: str
+    name: str
```

Comment:
Can you please add the line
```
Parameters
----------
```
to this docstring, just before listing all the arguments of the class constructor?